src/pages/host/VansList.jsx

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from "react";
+import Van from "../../components/cards/Van";
+import { Heading2, Heading3 } from "../../components/ui/Typography";
+
+const VansList = () => {
+  const [vansList, setVansList] = useState([]);
+  useEffect(() => {
+    fetch("/api/host/vans")
+      .then((resp) => resp.json())
+      .then((data) => setVansList(data.vans));
+  }, []);
+  return (
+    <div className="bg-[#fff7ed] w-full text-center">
+      <Heading2>Your listed Vans</Heading2>
+      {vansList ? (
+        <div className="w-full justify-center flex-col items-center flex gap-y-5 py-5">
+          {vansList.map((el) => (
+            <Van {...el} key={el.id} />
+          ))}
```

Comment:
You can also use this 
```suggestion
         {vansList.map((el) => <Van {...el} key={el.id} />)}
```
As you can see it has less lines.